docs/documentation_guide.md

```diff
@@ -0,0 +1,95 @@
+# Documentation Guide
+
+## Getting started with Fondant
+
+Learn about the Fondant project and how to get started with it.
+
+→ Start with the official guide on how to [install](installation.md) Fondant.  
+→ Get started by running your first fondant [pipeline](guides/first_pipeline.md) using the [local
+runner](runners/local.md). There you will learn how to use reusable components and how to build your
+own custom components.  
+→ Learn how to use the [data explorer](data_explorer.md) to explore the outputs of your pipeline.
+
+## Fondant fundamentals
+
+Learn how to use Fondant to build your own data processing pipeline.
+
+-> Design your own fondant [pipeline](pipeline.md) using the Fondant pipeline API.  
+-> Use existing [reusable components](components/hub.md) to build your pipeline.  
+-> Use [generic components](components/generic_component.md) to load/write your custom data format
+to/from Fondant.  
+-> Build your own [custom component](components/custom_component.md) using the Fondant component
+API.
+-> Learn how to publish your own [components](components/publishing_components.md) to a container
```

Comment:
I think you need two spaces at the end of the line above to make this one appear on a new line.